package.json

```diff
@@ -1,8 +1,20 @@
 {
-  "name": "wui-design",
-  "private": true,
-  "version": "0.0.0",
+  "name": "@waseem-polus/wui-design",
```

Comment:
try to find a better scope name for the package. @ wui org name is already taken on npm, so @wui/design is out :(